contracts/v2/PolygonZkEVMBridgeV2.sol

```diff
@@ -156,7 +156,7 @@ contract PolygonZkEVMBridgeV2 is
         IBasePolygonZkEVMGlobalExitRoot _globalExitRootManager,
         address _polygonRollupManager,
         bytes memory _gasTokenMetadata
-    ) public virtual initializer {
+    ) external virtual initializer {
```

Comment:
i know can be external, but thne if we import this bridge into some mock won't compile